utils/githooks/pre-commit.d/20-isort.sh

```diff
@@ -4,6 +4,13 @@
 
 set -ue
 
+# Disable isort check because it doesn't work
+# - it needs to run with -rc
```

Comment:
I used `--check-only` which is same as `-c`. but what is `-r`? cannot find that option.

Reply:
`-rc` is for recursive
```
$ isort --jobs 8 --check-only .
WARNING: Unable to parse file . due to [Errno 21] Is a directory: '/home/dbohning/daos'
```